### ISSUE TITLE
[BUG] `@html-eslint/attrs-newline` doesn't fix the expression value properly

### ISSUE THREAD

**Issue by dog-s-life** [yeonjuan/html-eslint#472]
Most of all, thank you for this good linting solution :)

**Describe the bug**

Expression value of the attribute in the template literal is removed when the `@html-eslint/attrs-newline` rule is applied.

**To Reproduce**

Watch the `foo` attribute on `<div>`

https://html-eslint.org/playground#eyJjb2RlIjoiaHRtbGBcbiAgICA8ZGl2IGEgYiBmb289JHtcInRydWVcIn0+XG4gICAgICAgIDxzcGFuPlxuICAgIDwvc3Bhbj5cbiAgICA8L2Rpdj5cbmBcblxuY29uc3QgaHRtbCA9IC8qaHRtbCovYFxuPGRpdj5cbiAgICA8c3Bhbj5cbiAgICAgICA8L3NwYW4+XG48L2Rpdj5cbmA7IiwiY29uZmlnIjp7InJ1bGVzIjp7IkBodG1sLWVzbGludC9pbmRlbnQiOiJlcnJvciIsIkBodG1sLWVzbGludC9hdHRycy1uZXdsaW5lIjoiZXJyb3IifX0sImxhbmd1YWdlIjoiamF2YXNjcmlwdCJ9

**Expected behavior**

`${"true"}` should not be removed

**Additional context**

https://github.com/yeonjuan/html-eslint/blob/59a64a0eeb70a3cf6d32866d276e62cbe30b36b6/packages/eslint-plugin/lib/rules/attrs-newline.js#L80

I guess this line is supposed to be

```js
            if (attr.value) {
                if (attr.startWrapper && attr.endWrapper) {
                  expected += `=${attr.startWrapper.value}${attr.value.value}${attr.endWrapper.value}`;
                } else {
                  expected += `=${attr.value.value}`;
                }
            }
```

because the expression doesn't have wrappers but value has entire string.

**Comment by yeonjuan** [comment]
@dog-s-life 
> because the expression doesn't have wrappers but value has entire string.

After checking, I can confirm that your analysis is correct.
I think it would be best to handle it as shown below. If you’d like to make the changes yourself, feel free to let me know. Otherwise, I can go ahead and update it.
```js
    if (attr.value) {
              const startWrapper = attr.startWrapper
                ? attr.startWrapper.value
                : "";
              const endWrapper = attr.endWrapper ? attr.endWrapper.value : "";
              expected += `=${startWrapper}${attr.value.value}${endWrapper}`;
            }
```

**Comment by dog-s-life** [comment]
@yeonjuan 
I also agree with your change! Thanks for quick update. 😃

**Comment by yeonjuan** [comment]
@dog-s-life It's fixed in `@html-eslint/eslint-plugin@0.52.1`!
